src/wp-admin/network/site-users.php

```diff
@@ -139,13 +139,27 @@
 
 		case 'promote':
 			check_admin_referer( 'bulk-users' );
+
+			if ( ! current_user_can( 'promote_users' ) ) {
```

Comment:
I don't think we need to add this check here. `get_editable_roles()` already provides a way to filter the list of roles a network administrator is able to assign and the `manage_sites` capability is already required to visit the `site-users.php` page.

Reply:
@jeremyfelt This check was made similar to the ones in the following file. Let me know, I should still remove the check.

```
File: src/wp-admin/users.php
111: 		check_admin_referer( 'bulk-users' );
112: 
113: 		if ( ! current_user_can( 'promote_users' ) ) {
114: 			wp_die( __( 'Sorry, you are not allowed to edit this user.' ), 403 );
115: 		}
```